libs/ng-strapi-auth/src/lib/core/services/auth/auth.service.ts

```diff
@@ -1,9 +1,69 @@
-import { Injectable } from '@angular/core';
+import { Inject, Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { BehaviorSubject, Subject } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { map } from 'rxjs/operators';
+import { LocalStorageKeyEnum, UserModel } from '@ng-strapi-auth/ng-strapi-auth';
+import { AuthOptionModel } from '../../../ng-strapi-auth-options';
 
 @Injectable({
-  providedIn: 'root'
+    providedIn: 'root'
 })
 export class AuthService {
 
-  constructor() { }
+    authToken: any;
+    authApiBase: string;
+    authUserChanged$: Subject<UserModel | null>;
+
+    constructor(
+        private httpClient: HttpClient,
+        @Inject('StrapiAuthLibOptions') private readonly options: AuthOptionModel,
+        public router: Router,
+    ) {
+        this.authApiBase = this.options.baseAPIPath;
+        this.getUserFromServer().subscribe((user) => {
```

Comment:
Check if we have a JWT. If we don't, just set the authUserChanged to false / null.
Also use Subject and not behavior subject